pkg/controller/podgroup.go

```diff
@@ -102,7 +102,7 @@ func (ctrl *PodGroupController) Run(workers int, stopCh <-chan struct{}) {
 	}
 	klog.Info("Coscheduling sync finished")
 	for i := 0; i < workers; i++ {
-		go wait.Until(ctrl.sync, 0, stopCh)
+		go wait.Until(ctrl.worker, 0, stopCh)
```

Comment:
could you also make the logic consistent with eq controller?

- use `second` instead of 0 here
- processNextWorkItem returns true on abnormal cases
- return in worker()

Reply:
Sure